test/scaling/dimspecs.jl

```diff
@@ -0,0 +1,26 @@
+module ScalingDimspecTests
+
+using Interpolations, DualNumbers, Base.Test
+
+xs = -pi:(2pi/10):pi-2pi/10
+ys = -2:.1:2
+f(x,y) = sin(x) * y^2
+
+itp = interpolate(Float64[f(x,y) for x in xs, y in ys], Tuple{BSpline(Quadratic(Periodic)), BSpline(Linear)}, OnGrid)
+sitp = scale(itp, xs, ys)
+
+# Don't test too near the edges until issue #64 is resolved
+for (ix,x0) in enumerate(xs[5:end-5]), (iy,y0) in enumerate(ys[2:end-1])
+    x, y = x0 + 2pi/20, y0 + .05
+    @test_approx_eq sitp[x0, y0] f(x0,y0)
+    @test_approx_eq_eps sitp[x0, y0] f(x0,y0) 0.05
```

Comment:
I think this is actually residue from trying to figure out good ways to test the scaling. I ended up doing just one test, on the grid points, since if that works then it's reasonably safe to assume that the coordinate scaling has worked correctly. Interpolating between grid points is then tested elsewhere.